queries.md

```diff
@@ -2,18 +2,38 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+select * from customers
+where city is 'London'
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+select * from customers
+where postalCode is 1010
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+select *, supplierId as id from suppliers
+where id = 11
```

Comment:
Whenever you use `*` with aliases, it'll replicate the columns which aren't named in the current query. That means you have a `supplierId` column and an `id` column that are identical with this query. It's best to either take them all in (`*`) or be explicit by pulling in what you need by name. It also looks like it's asking for you to find the phone number for this supplier, not all of the details!